plugins/modules/pagerduty_alert.py

```diff
@@ -53,11 +55,37 @@
             - 'triggered'
             - 'acknowledged'
             - 'resolved'
-    api_key:
+    api_version:
         type: str
         description:
-            - The pagerduty API key (readonly access), generated on the pagerduty site.
-        required: true
+            - The Api version we want to use to run the module.
+            - V1 is more limited with option we can provide to trigger incident
+            - V2 has more variables for example, severity, source, custom_detail,etc.
```

Comment:
```suggestion
            - V2 has more variables for example, I(severity), I(source), and I(custom_detail).
```
Use correct formatting to reference to other options (https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation).